config/settings/production.py

```diff
@@ -115,6 +115,14 @@
 AWS_PRELOAD_METADATA = True
 COLLECTFAST_STRATEGY = "collectfast.strategies.filesystem.FileSystemStrategy"
 
+# FILE UPLOAD SIZING
+# Useful settings to configure for GraphQL actions that upload large files like base64 encoded blobs
+# Default values for both are 2.5MiB. The options below up the values to 10MiB.
+# https://docs.djangoproject.com/en/1.11/ref/settings/#file-upload-max-memory-size
+FILE_UPLOAD_MAX_MEMORY_SIZE = 10485760
```

Comment:
From the docs, `FILE_UPLOAD_MAX_MEMORY_SIZE` is just the limit before the body spills over from memory to the filesystem, so it shouldn't need to be increased.

`DATA_UPLOAD_MAX_MEMORY_SIZE` looks like the actual limit.

Reply:
The check is baked into the `HttpRequest.body` attribute. However we can bypass it for this endpoint in specific by using `HttpRequest.read` (and also `close`).